pkg/rootless/rootless_linux.c

```diff
@@ -232,6 +253,13 @@ reexec_in_user_namespace (int ready)
       _exit (EXIT_FAILURE);
     }
 
+  if (chdir (cwd) < 0)
+    {
+      fprintf (stderr, "cannot chdir: %s\n", strerror (errno));
+      _exit (EXIT_FAILURE);
+    }
+  free (cwd);
```

Comment:
Nit, after fixing this up in fuse-overlay. Shouldn't this be cleaned up automatically.

Reply:
we could start using auto cleanups, although this doesn't really make any practical difference as we are going to `exec`